torchvision/csrc/io/decoder/stream.cpp

```diff
@@ -65,13 +65,18 @@ int Stream::openCodec(std::vector<DecoderMetadata>* metadata, int num_threads) {
     // otherwise set sensible defaults
     // with the special case for the different MPEG4 codecs
     // that don't have threading context functions
+#if LIBAVCODEC_VERSION_MAJOR >= 60
```

Comment:
- Check if it's defined first.
- Should we just wrap the branch condition, and simply say `false` when >= 60?